test/utils/ui.test.helper.ts

```diff
@@ -77,3 +120,31 @@ export const fillVisibleInput = async (
   await page.waitForSelector(selector, { hidden: false, visible: true });
   await page.type(selector, value);
 };
+
+/**
+ * Returns 3 values:
+ ** 1st: if email doesn't contain link - the message
+ ** 2nd: if email contains URL - the link
+ ** 3rd: always returns number of all emails
+ * @param selector of coockies
+ */
+export const getEmails = async () => {
+  let response = await restRequestAuth(TestUser.GLOBAL_ADMIN);
+  let emails = response.body.mailItems[0].body;
+
+  function detectUrl(text: string) {
+    let cleanText = text.replace(/<.*?>/gm, '');
+    let urlRegex = /(((https?:\/\/)|(https:\/\/)|(www\.))[^\s]+)/g;
+    let url = cleanText.match(urlRegex);
+    return url?.toString();
+  }
+
+  let url = detectUrl(emails);
+  let emailsCount = response.body.totalRecords;
+
+  if (!url) {
+    return [emails, emailsCount];
+  }
+
+  return [url, emailsCount];
+};
```

Comment:
The documentation states `3 values` but I see only two returned.
It's not a good practice to return different things - return ALWAYS 3, but have one `undefined`.
You can also define the return type as the above comment (the exact type is up to you, it's just what I understood from the code).